app/controllers/reservations_controller.rb

```diff
@@ -35,6 +35,28 @@ def handle_pending_reservation_admin
     end
   end
 
+  # create my reservations for a user
+  def my_reservations
+    return render json: { error: 'You need to loginto access this page' }, status: :unauthorized unless logged_in?
+
+    @reservations = Reservation.find(user_id: 25)
+    raise ActiveRecord::RecordNotFound unless @reservations
+
+    render json: @reservations, each_serializer: ReservationSerializer, status: :ok
+  end
+
+  # delete reservation
+  def delete_reservation
+    if logged_in?
+      @reservation = current_user.reservations.find(params[:reservation_id])
+      raise ActiveRecord::RecordNotFound unless @reservation
+
+      destroy_reservation(@reservation)
+    else
+      render json: { error: 'You need to loginto access this page' }, status: :unauthorized
```

Comment:
Make sure to fix the spelling mistake in this message. `loginto`

Reply:
@aimalamiri  Done